lib/queue.js

```diff
@@ -309,6 +359,26 @@ export default class Queue {
     }).then(() => {
       return id;
     });
+  }
+
+  sendDelayedMessages() {
+    const query = {
+      status: 'delayed',
+      sendAfter: { $lte : new Date() }
+    }
+
+    var deferred = defer();
```

Comment:
Actually you should just do a bulk update of the collection to set all docs with `_id: { $in: [ids] } ` to `status:'queued'` and then it's only one update.